### PR TITLE
Remove use of Client Secret

### PR DESCRIPTION
**Pull request by ambarwootric**
No longer need to use client secret to intialize sdk.

Test: Run demo app just by updating client id and account token (no more client secret is needed) and see if everythings run smoothly.

### CODE REVIEW

**Comment by matchbzadmin** [comment]
Sounds good @diegoserranoa, ok I will take a look into and see how I can make both works with new method addition. Thanks for your review and suggestion.

**Comment by ambarwootric** [comment]
Sounds good @diegoserranoa, ok I will take a look into and see how I can make both works with new method addition. Thanks for your review and suggestion.

**Comment by ambarwootric** [comment]
@diegoserranoa I have tried to add new method and update related codes, let me know more about the changes!

File: androidsdk/src/main/java/com/wootric/androidsdk/network/tasks/GetAccessTokenTask.java [hunk 1]
```diff
@@ -50,7 +50,9 @@ protected String requestUrl() {
     protected void buildParams() {
         paramsMap.put("grant_type", "client_credentials");
         paramsMap.put("client_id", clientId);
-        paramsMap.put("client_secret", clientSecret);
+        if (clientSecret != null) {
```
Comment (prabhatjha) [inline androidsdk/src/main/java/com/wootric/androidsdk/network/tasks/GetAccessTokenTask.java]: I don't think we should even use `client_secret` for best security practices. @diegoserranoa what do you think?

Comment (ambarwootric) [reply androidsdk/src/main/java/com/wootric/androidsdk/network/tasks/GetAccessTokenTask.java]: Umm yes, but for current use, we are trying to have both. But while using `client_secret` user will get `depreciated` warning. Ok let's see what @diegoserranoa says

Comment (diegoserranoa) [reply androidsdk/src/main/java/com/wootric/androidsdk/network/tasks/GetAccessTokenTask.java]: @prabhatjha @ambarwootric I was trying to go for the approach of a slow transition so users can still use this method. If we want user to completely use the new method, then you should probably just go to adding the new method @ambarwootric.

Comment (prabhatjha) [reply androidsdk/src/main/java/com/wootric/androidsdk/network/tasks/GetAccessTokenTask.java]: @diegoserranoa ok, let's go with your approach. I guess it's ready to ship then.

**Comment by ambarwootric** [comment]
Thanks for such a good review and your time @diegoserranoa. Also thanks to @prabhatjha for getting into this.

**Pull request merged**